ios/Podfile

```diff
@@ -43,11 +63,21 @@ target 'gutenberg' do
   pod 'Gutenberg', :path => '../Gutenberg.podspec'
 
   target 'gutenbergTests' do
-    inherit! :search_paths
+    inherit! :complete
     # Pods for testing
   end
 
   use_native_modules!
+
+  # Enables Flipper.
+  #
+  # Note that if you have use_frameworks! enabled, Flipper will not work and
```

Comment:
Hey @SergioEstevao 👋 
please, can you take a look at this and see if we can enable Flipper? 
(as it seems that we have use_frameworks! enabled?) Thanks.

Reply:
Last time I tried on the beta versiosn we couldn't, I can give it a go again if you want?

Reply:
Well if you think that it doesn't make a sense then we can leave it as it is and try in the future or if there is some other way to enable it?